README.md

```diff
@@ -842,7 +853,9 @@ Valid keys and their meanings:
 
 * **write_to_cache** - Accepts values `true` or `false`.  If `false`, the completed calls from this workflow will not be added to the cache.  See the [Call Caching](#call-caching) section for more details.
 * **read_from_cache** - Accepts values `true` or `false`.  If `false`, Cromwell will not search the cache when invoking a call (i.e. every call will be executed unconditionally).  See the [Call Caching](#call-caching) section for more details.
+* **workflow_log_dir** - Specifies a path where per-workflow logs will be written.  If this is not specified, per-workflow logs will not be copied out of the Cromwell workflow log temporary directory/path before they are deleted.
 * **outputs_path** - Specifies a path where final workflow outputs will be written.  If this is not specified, workflow outputs will not be copied out of the Cromwell workflow execution directory/path.
+* **call_logs_dir** - Specifies a path where final call logs will be written.  If this is not specified, call logs will not be copied out of the Cromwell workflow execution directory/path.
```

Comment:
@scottfrazer Re:

> If this is not specified, call logs will not be copied out of the Cromwell workflow execution directory/path.

Let me know if this is still not clear enough. The sentence was mostly copied from the one above, but happy to have it reworded to be clearer. Also I tested basic functionality of copying to/from gcs and local, so let me know if you see unexpected behavior?